File: solutions/0168_Excel_Sheet_Column_Title/iter.py
class Solution:
    def convertToTitle(self, n: int) -> str:
        ret = ''
        while n > 0:
            ret += chr((n-1)%26 + 65)
            n = (n-1) // 26 # -1 is the mvp
        return ret[::-1]

File: solutions/0191_Number_of_1_Bits/iter.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        int ret = 0;
        while (n > 0) {
            ret += (n & 1) > 0 ? 1 : 0;
            n >>= 1;
        }
        return ret;
    }
};
